application/backend/src/main/resources/libs/bulma/components/title/dialog/.content.json

```diff
@@ -38,14 +21,37 @@
           "name": "element"
         }
       },
-      "isSpaced": {
-        "sling:resourceType": "wcm/dialogs/components/checkbox",
-        "name": "isSpaced",
-        "label": "Normal spacing between title and subtitle",
+      "subtitleOptionsContainer": {
+        "sling:resourceType": "wcm/dialogs/components/container",
+        "addSubtitle": {
+          "sling:resourceType": "wcm/dialogs/components/toggle",
+          "name": "addSubtitle",
+          "label": "Add a subtitle"
+        },
+        "subtitleContainer": {
+          "sling:resourceType": "wcm/dialogs/components/container",
+          "subtitle": {
+            "sling:resourceType": "wcm/dialogs/components/textfield",
+            "name": "subtitle",
+            "label": "Subtitle text",
+            "description": "A subtitle can be defined if the size of the title is 4 or smaller"
```

Comment:
Shouldn't it be `or bigger` instead of `or smaller`? Smaller number is a bigger size :)

Reply:
Yeah, that is really confusing and i thought that if we talk about numbers, subtitle can be defined for NUMBER 4 and smaller, so for 3,2 and 1.
Do you still want me to change the text?
This reverse logic is so bad 😢

Reply:
Yes, I think it was also specified in AC as `4 or bigger`?

Reply:
Fixed.